### PR TITLE
move AccountSelector to Selector

### PR DESCRIPTION
**Pull request by mazmassa**
⚠️ This is a breaking change ⚠️

We are moving `<AccountSelector>` to `<Selector>` and update the API to fit our need since this component is not only linked to Accounts.

These are the updates:

Component:

 - `<AccountSelector>` -> `<Selector>`

Properties:
 - avatar -> preIcon
 - accountName -> content
- icon -> posIcon

We also don't need to pass any class to the preIcon or/and  posIcon once the component manages now the sizes.

Screenshot 2023-06-01 at 01 25 55

### CODE REVIEW

File: src/components/Selector/Selector.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    content,
+    posIcon,
+    amount,
+    variant = 'primary',
+    children,
+    customClass,
+    ...rest
+  } = props;
+
+  const classes: any = {
```
Comment (Thykof) [inline src/components/Selector/Selector.tsx:29]: could you create an interface for this?

File: src/components/Selector/Selector.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+const args = {
+  preIcon: <FiArrowUpRight />,
+  content: 'account #',
+  posIcon: <MassaLogo size={24} />,
+  amount: '0,000.00',
+};
+
+export const _Primary = {
+  render: () => <Selector {...args} customClass="!w-1/2" />,
```
Comment (maxime-carabina) [inline src/components/Selector/Selector.stories.tsx:17]: `!` is necessary ?

Comment (mazmassa) [reply src/components/Selector/Selector.stories.tsx:17]: Mystery: for some class yes, for others no! With `w-1/3` no, with `w-1/2` yes... Anyway I left there to emphasise that if it does not take in account the !(**!important**) will make it.

File: src/components/Selector/Selector.tsx (hunk 1 above)
Comment (mazmassa) [reply src/components/Selector/Selector.tsx]: Ohh yes! Done!

**Pull request merged**